src/components/PictureGallery/PictureGallery.js

```diff
@@ -75,6 +131,22 @@ const SectionTitle = styled.div`
   grid-column: 1 / -1;
 `;
 
+const ImageDateContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+`;
+
+const DateTitle = styled.div`
+  font-size: 16px;
+  line-height: 19px;
+  color: black;
+  margin-top: 10px;
```

Comment:
Can you move this to the top? (line 142)
We try to follow [idiomatic css](https://github.com/necolas/idiomatic-css) guidelines